### ISSUE TITLE
Docker 报错

### ISSUE THREAD

**Issue by zjx-kimi** [QingdaoU/OnlineJudgeDeploy#107]
```
root@admin:/home/kimi/OnlineJudgeDeploy# sudo docker-compose up -d
service "oj-backend" depends on undefined service judge-server: invalid compose project
```

**Comment by Dorggie** [comment]
It seems like typo. Change  depends_on: judge-server --> oj-judge

I also switch the image to original one:
oj-judge:
    image: registry.cn-hangzhou.aliyuncs.com/onlinejudge/judge_server
oj-backend:
    image: registry.cn-hangzhou.aliyuncs.com/onlinejudge/oj_backend

It works for me. Hope it helps!

**Comment by mikucat0309** [comment]
Fixed in f98842c67ec25d30641de767a895f2aafeca6e96
